src/components/ui/StaticFileMetadata.vue

```diff
@@ -67,8 +69,14 @@ export default class StaticFileMetadata extends Vue {
     return !!this.ffaListing ? this.ffaListing.license : ''
   }
 
-  public get fileSize(): number {
-    return !!this.ffaListing ? this.ffaListing.size : 0
+  public get fileSize(): string {
+    if (!this.ffaListing) { return '' }
+    return FileHelper.fileSizeString(this.ffaListing.size)
+  }
+
+  public get costETH(): string {
+    if (!this.ffaListing) { return '' }
+    return FileHelper.costString(this.ffaListing.size, this.appModule.costPerByte)
```

Comment:
`return !this.ffaListing ? '' : FileHelper.costString(this.ffaListing.size, this.appModule.costPerByte)`